client/card-readers/settings/sections/branding-details.js

```diff
@@ -84,7 +84,7 @@ const BrandingDetailsSection = () => {
 					'woocommerce-payments'
 				) }
 				purpose="business_icon"
-				value={ getAccountBrandingIcon }
+				fileID={ getAccountBrandingIcon }
```

Comment:
Would it be possible to the user a hint of what is going on here with a spinner? I mean while the Icon and logo are being retrieved. With the current behavior, the image 'pops' in when it is ready. Maybe this is a matter of preference but I think would be nice. I also don't know if we should address it in a different issue.

Also we could handle the error while retrieving the file. Currently it shows a broken image:
<img width="576" alt="Screen Shot 2022-02-07 at 11 38 31" src="https://user-images.githubusercontent.com/1553182/152772636-e0323f7c-a22b-4dfb-b816-916c0bc72d33.png">

Reply:
I agree with Miguel, at least on local environment the upload time is significant enough to make some action

Reply:
> Would it be possible to give the user a hint of what is going on here with a spinner?
During the upload, the upload button has an animation which indicates that the file is being uploaded. Indeed, during the upload, we can do some other actions, like pressing the save button. 
We could create a new issue to disable the save button until the file is fully uploaded. 
What's your opinion on this ?

Reply:
> We could create a new issue to disable the save button until the file is fully uploaded.
What's your opinion on this ?

Yep, I think is a nice improvement, we can tackle it in a different issue.

Reply:
> During the upload, the upload button has an animation which indicates that the file is being uploaded
I am afraid that my original comment remains though, since I am referring to the download of the image when the page loads, for instance.